app/derivative_services/av_derivative_service.rb

```diff
@@ -41,7 +41,7 @@ def change_set
   end
 
   def valid?
-    target_file && ["audio/x-wav", "audio/vnd.wave"].include?(mime_type.first)
+    target_file && ["audio/x-wav", "audio/vnd.wave", "video/mp4"].include?(mime_type.first)
```

Comment:
This list seems similar to but not exactly the same as the configured mediainfo mime types?